src/fidgets/ui/Text.tsx

```diff
@@ -91,6 +114,10 @@ export const Text: React.FC<FidgetArgs<TextFidgetSettings>> = ({
       style={{
         background: settings.background,
         height: "100%",
+        borderWidth: settings.fidgetBorderWidth,
+        borderColor: settings.fidgetBorderColor,
```

Comment:
Can we apply these styles to the fidget wrapper card as well so they affect all fidgets?

Reply:
Updated! ty!